routes/stories.js

```diff
@@ -117,7 +117,10 @@ router.get(
         },
       });
     }
-
+    //sort comments by newest
+    story.Comments.sort((b,a) => {
+     return a.createdAt.getTime() - b.createdAt.getTime();
+    });
```

Comment:
This should be do-able in the query. Something like:

```js
const story = await Post.findByPk(storyId, {
  include: [
    { model: Comment, include: [CommentLike, User] },
    PostLike,
    { model: User, include: [{ model: User, as: "Followers" }] },
  ],
  order: [
    [Post, Comment, 'createdAt', 'DESC'],
  ],
});
```

Reply:
this query breaks the rest of the code